nimbus.fml.yaml

```diff
@@ -68,8 +68,220 @@ features:
         description: Where is the message to be put.
         type: Option<MessageSurfaceId>
         default: null
+  messaging:
+    description: |
+      Configuration for the messaging system.
+
+      In practice this is a set of growable lookup tables for the
+      message controller to piece together.
+
+    variables:
+      messages:
+        description: A growable collection of messages
+        type: Map<String, MessageData>
+        default: {}
+
+      triggers:
+        description: >
+          A collection of out the box trigger
+          expressions. Each entry maps to a
+          valid JEXL expression.
+        type: Map<String, String>
+        default:
+          english-speaking: "'en' in locale"
+          core-active: "count(daysOfUse) > 25"
+
+      styles:
+        description: >
+          A map of styles to configure message
+          appearance.
+        type: Map<String, StyleData>
+        default:
+          urgent:
+            background-color: red
+            text-color: white
+            button-background: bright-blue
+            button-text-color: white
+            priority: 70
+          warning:
+            background-color: cyan,
+            text-color: black,
+            button-background: bright-blue
+            button-text-color: white
+            priority: 55
+          default:
+            background-color: blue
+            text-color: white
+            button-background: bright-blue
+            button-text-color: white
+            priority: 50
+          excited:
+            background-color: blue
+            text-color: white
+            button-background: bright-blue
+            button-text-color: white
+            priority: 60
+
+      actions:
+        type: Map<String, String>
+        description: A growable map of action URLs.
+        default:
+          OPEN_SETTINGS: firefox://settings
+          OPEN_SYNC_SETTINGS: firefox://settings/sync
+          OPEN_POCKET_SETTINGS: https:///getpocket.com/settings?fxa={fxa-token}
```

Comment:
@jhugman just to make you aware the scheme for deeplink changes depending of on which build type (nightly,beta, and release)
nightly -> fenix-nightly://
beta -> fenix-beta://
release -> fenix://

See https://github.com/mozilla-mobile/fenix/blob/main/app/build.gradle#L113-L137